packages/vue-components/components/commons/generic-form/generic-form.vue

```diff
@@ -38,7 +54,79 @@
               toggleMask
               :feedback="false"
             />
-            <PrimeMessage severity="error" v-else>
+            <!-- number / float -->
+            <PrimeInputNumber
+              v-else-if="f.component === FIELD_COMPONENT.INPUT_NUMBER"
+              :name="f.id"
+              :class="f.class"
+              fluid
+            />
+            <!-- date / datetime -->
+            <PrimeInputText
+              v-else-if="
+                [FIELD_COMPONENT.INPUT_DATE, FIELD_COMPONENT.INPUT_DATETIME].includes(f.component)
+              "
+              :name="f.id"
+              :class="f.class"
+              fluid
+              show-icon
+              icon-display="input"
+              append-to="body"
+              :show-time="f.component === FIELD_COMPONENT.INPUT_DATETIME"
+              :type="f.component === FIELD_COMPONENT.INPUT_DATE ? 'date' : 'datetime-local'"
+            />
+            <!-- single select -->
+            <PrimeSelect
+              v-else-if="f.component === FIELD_COMPONENT.SINGLE_SELECT"
+              :name="f.id"
+              :class="f.class"
```

Comment:
```suggestion
              :class="f.class"
              :labelId="f.id"
```

If I understand the documentation, it's `labelId` now to attribute an `id` to the `<select>`, but you should check cause I'm not really sure.